src/lib/math.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * © 2024 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const src = 'https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-svg.js'
+
+/**
+ * MathJax uses delimiters to find math in the doc by default, which is
+ * not what we want. This will override that behavior and will instead
+ * look for equation nodes, and is based on the
+ * [MathJax upgrade guide](https://docs.mathjax.org/en/v3.2-latest/upgrading/v2.html).
+ */
+//@ts-ignore
+const findMath = (doc) => {
+  let nodes = doc.options.elements
+  if (!nodes) {
+    nodes = doc.document.querySelectorAll('.equation')
+  }
+  for (const node of nodes) {
+    const math = new doc.options.MathItem(
+      node.textContent,
+      doc.inputJax[0],
+      node.tagName !== 'SPAN'
+    )
+    const text = document.createTextNode('')
+    node.innerHTML = ''
+    node.appendChild(text)
+    math.start = { node: text, delim: '', n: 0 }
+    math.end = { node: text, delim: '', n: 0 }
+    doc.math.push(math)
+  }
+}
+
+const config = {
+  options: {
+    renderActions: {
+      find: [10, findMath, ''],
+    },
+  },
+}
+
+export const initMathJax = () => {
+  // @ts-ignore
```

Comment:
Is the mathjax lib loaded via <script> smaller than the code that we load into our bundle? Can we create a separate bundle for mathjax during our build process and load it here instead of loading it from cdn? My only concern is that we make it a bit of a hidden thing that we use mathjax and that we use that version of it, because now if you'll check package json it's not there, although our project depends on it. So generation of our local separate bundle for that maybe more work, but maybe a better approach in terms of dependencies management.

Reply:
It's a bit smaller, yes. However, the main benefit of this way compared to the old way is that the mathjax script can be cached by the browser. 

It is possible to use NPM and (host our own copy)[https://www.npmjs.com/package/mathjax#hosting-your-own-copy-of-the-mathjax-components] if that's preferred, but I'll have to look into how this interacts with vite.